simplq/src/components/common/SidePanel/SidePanelItem.jsx

```diff
@@ -33,17 +27,30 @@ const SidePanelItem = ({
   children,
 }) => {
   const [open, setOpen] = useState(false);
-  const toggleOpen = () => setOpen(!open);
+  const onClickHandler = () => {
+    if (expandable) {
+      setOpen(!open);
+    }
+    if (onClick) {
+      onClick();
+    }
+  };
 
   return (
-    <button type="button" className={styles['side-panel-item']} onClick={onClick}>
```

Comment:
remove button, as it a nasty react warning when there is a button inside a button, which we have.